src/components/TodoForm.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+class TodoForm extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            taskItem: ''
+        };
+    }
+
+    handleChanges = e => {
+        this.setState({ taskItem: e.target.value });
+    };
+
+    handleAddTodo = e => {
+        e.preventDefault();
+        this.props.addTodo(this.state.taskItem);
```

Comment:
Question: What could you do to clear the form after a todo item is submitted? Message me if you need more of a hint.